tests/driver.sh

```diff
@@ -1647,6 +1647,12 @@ int main() {
 }
 EOF
 
+# The following cases validate the behavior and return value of
+# snprintf(). Notice that you should refer to the specification,
+# such as C99 7.19.6.5, to understand the behavior of snprintf().
```

Comment:
Instead of saying "you should refer to," quote the essential contents from C standards if you want to emphasize.